### DISCUSSION TITLE
buddhism/2024/%E6%B3%95%E7%84%A1/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#163] [General]
# buddhism/2024/%E6%B3%95%E7%84%A1/

https://shanenull.com/buddhism/2024/%E6%B3%95%E7%84%A1/

**Comment by shane0** [comment]
# Since there is Buddha nature, why did you crash into this stinky skin bag? 

- lol http://chan.bailinsi.net/2016/6/2016613.htm

**Reply by shane0** [reply]
https://terebess.hu/zen/YouruWang.html

**Reply by shane0** [reply]
- 36 pairs https://www.judithragir.org/2011/07/the-opposites-of-huineng/

**Reply by shane0** [reply]
# 36 pairs

The concept of the **Thirty-Six Pairs of Dharma** is not a universally defined or standardized list across all Buddhist traditions, so it may not be explicitly named in many sutras. However, there are texts that refer to complementary or paired principles that could be seen as analogous to the idea of "Thirty-Six Pairs."

A few important sutras and teachings where you may find complementary concepts that could be understood as pairs of Dharma include:

### 1. **The *Mahāparinirvāṇa Sūtra* (大般涅槃經)**
   - This sutra, one of the important texts in Mahayana Buddhism, often deals with paired concepts, such as the relation between form and emptiness (śūnyatā), truth and illusion, or wisdom and compassion. These could be interpreted as complementary pairs.

### 2. **The *Vimalakīrti Nirdesa Sūtra* (維摩詰所說經)**
   - This Mahayana sutra features teachings on the non-duality of opposites, with pairs like samsara and nirvana, or appearance and emptiness. The Vimalakīrti Sūtra is often cited for its emphasis on the simultaneous understanding of conventional truth and ultimate truth.

### 3. **The *Lotus Sūtra* (法華經)**
   - In the Lotus Sūtra, there is a recurring theme of the unity of opposites, such as the parable of the raft, which teaches the distinction between form and emptiness or the Buddha’s skillful means (upāya) and the ultimate reality. The Lotus Sūtra often presents ideas in paired or dualistic frameworks.

### 4. **The *Diamond Sūtra* (金剛經)**
   - In this text, the Buddha repeatedly addresses the relationship between form and emptiness, offering insights into how seemingly contradictory concepts—like existence and non-existence—are ultimately interdependent, revealing a deeper truth.

### 5. **The *Heart Sūtra* (般若波羅蜜多心經)**
   - The Heart Sutra is famous for its teaching on the non-duality of form and emptiness, which could be seen as one of the pairs of Dharma. It encapsulates the paradoxical nature of ultimate reality.

### 6. **The *Dhammapada* (法句經)**
   - While not directly listing "pairs of Dharma," this early text in the Theravada tradition presents many aphorisms in pairs, such as the contrast between wisdom and ignorance, good and evil, or right and wrong.

### 7. **The *Avataṃsaka Sūtra* (華嚴經)**
   - The *Avataṃsaka Sūtra* often discusses the interconnectedness of all things in a way that emphasizes pairs or dualities, such as the relationship between the one and the many, the worldly and the transcendent.

### Potential Interpretations of "Thirty-Six Pairs of Dharma"
While there is no direct, universally agreed-upon list of the "Thirty-Six Pairs of Dharma" in primary Buddhist texts, scholars and teachers sometimes pair teachings that are complementary or oppositional. Some of these might include:

1. **Form and Emptiness**
2. **Samsara and Nirvana**
3. **Compassion and Wisdom**
4. **Impermanence and Nirvana**
5. **Skillful Means and Wisdom**
6. **Right View and Right Action**

Thus, the concept of Thirty-Six Pairs could be a way of conceptualizing these oppositional yet complementary principles found in many Mahayana and Theravada texts.

If you are seeking a specific set of pairs from a particular tradition or text, or have more specific details about where you encountered the term, I can help refine the list!

**Reply by shane0** [reply]
https://www.judithragir.org/2011/07/the-opposites-of-huineng/

**Comment by shane0** [comment]
# shrodinger’s cat’s bird

> A wise old man often solved various difficult problems for the villagers and was deeply respected by the people. One day, a clever and naughty child wanted to deliberately embarrass the old man. He caught a bird, held it in his palm, and ran to ask the old man: "Grandpa, I heard that you are the wisest man, but I don't believe it. If you can guess whether the bird in my hand is alive or dead, I will believe it." The old man looked at the child's cunning eyes, smiled and said: "It's a dead bird." The child let go of his hand, let the live bird fly away, and danced with joy for being able to win the wise old man.
>> In fact, the old man knew very well that if he answered that the bird was alive, the child would secretly strangle the bird to death; only if he answered that it was dead, the child would open his arms and let the bird fly away. Those who say that there is no fixed saying are wise; there is no fixed law, which is the wisdom of Buddhism. As the Diamond Sutra says: "As I understand the meaning of what the Buddha said, there is no fixed law, which is called Anuttara Samyaksambodhi; there is also no fixed law that the Tathagata can say."

**Comment by shane0** [comment]
# joshu’s dog

…
It is not nothingness, the opposite of existence.
If you really want to pass this barrier, you should
feel like drinking a hot iron ball that you can
neither swallow nor spit out.
Then your previous lesser knowledge disappears.
As a fruit ripening in season, your subjectivity and
objectivity naturally become one. It is like a dumb
manwhohas had a dream. Heknows aboutit buthe
cannot tell it.
When he enters this condition his ego-shell is
crushed and he can shake the heaven and move the
earth. Heis like a great warrior with a sharp sword.
If a Buddha stands in his way, he will cut him
down; if a patriarch offers him any obstacle, he
will kill him; and he will be free in his way of
birth and death. He can enter any world as if it
were his own playground. I will tell you how to
do this with this koan:
Just concentrate your whole energy into this
Mu, and do not allow any discontinuation. When
you enter this Mu and there is no discontinuation,
your attainment will be as a candle burning and
illuminating the whole universe.
Has a dog Buddha-nature?
This is the most serious question of all.
Ifyou say jes or no,
You loseyour own Buddha-nature.
